examples/org.eclipse.xtext.xtext.ui.examples/contents/org.eclipse.xtext.example.domainmodel.tests/.settings/org.eclipse.jdt.launching.prefs

```diff
@@ -0,0 +1,3 @@
+#Wed May 17 20:17:17 CEST 2016
```

Comment:
It's to avoid the warning about missing strictly compatible JRE. This file is already present in the domainmodel and domainmodel.ui projects, so I think it's good to have it also in the .tests project.